src/Elmish.WPF/ViewModel.fs

```diff
@@ -357,25 +348,22 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | Cmd _
     | CmdParam _ ->
         false
-    | SubModel ((vm: ViewModel<obj, obj> voption ref), (getModel: 'model -> obj voption), getBindings, toMsg, sticky) ->
+    | SubModel (vm, getModel, getBindings, toMsg, sticky) ->
       match !vm, getModel newModel with
       | ValueNone, ValueNone -> false
       | ValueSome _, ValueNone ->
-          if sticky then false
+          if sticky
+          then false
           else
             vm := ValueNone
             true
       | ValueNone, ValueSome m ->
-          vm := ValueSome <| ViewModel(
-            m, toMsg >> dispatch, getBindings (), config, getPropChainFor bindingName)
+          vm := ValueSome <| ViewModel(m, toMsg >> dispatch, getBindings (), config, getPropChainFor bindingName)
           true
       | ValueSome vm, ValueSome m ->
           vm.UpdateModel m
           false
-    | SubModelWin
-        ( vmWinState, getState, getBindings, toMsg,
-          getWindow, isModal, onCloseRequested, preventClose, winRef) ->
-
+    | SubModelWin (vmWinState, getState, getBindings, toMsg, getWindow, isModal, onCloseRequested, preventClose, winRef) ->
```

Comment:
Line's getting a bit long, but if you think it's better, I can accept it

Reply:
I prefer to think that this line is long not because we didn't include a rather arbitrarily placed new line but because `SubModelWin` has so many arguments.  When I am in the area, I will keep my eyes open for ways that we can reduce the number of arguments.

Reply:
Sure, though there's nothing wrong with splitting arguments across lines. I sometimes do, as you can see here and there.